edkrepo_manifest_parser/edk_manifest.py

```diff
@@ -380,6 +396,30 @@ def get_combo_element(self, name):
                 return copy.deepcopy(combo)
         raise ValueError(COMB_UNKOWN_ERROR.format(name, self._fileref))
 
+    def get_all_patchsets(self):

```

Comment:
Lets make this a property 

Per the issue this should return a list. Currently it is returning a dict. We should do something like the following:
`patchsets = []`
`for patch in self._patch_sets.keys():`
     `   patchsets.append(self._patch_sets[patch]`
`return patchsets`

Reply:
Please add the @property decorator to this method